dso/model.py

```diff
@@ -24,6 +24,7 @@ class ScanArtifact:
 class Datasource:
     ARTEFACT_ENUMERATOR = 'artefact-enumerator'
     BDBA = 'bdba'
+    CM06 = 'cm06'
```

Comment:
Please also add handling for `CM06` datasource in `datasource_to_datatypes` function.